modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/HistoricalSnapshotOrdinalEntries.scala

```diff
@@ -0,0 +1,79 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.data.NonEmptyList
+import cats.syntax.all._
+
+import scala.util.control.NoStackTrace
+
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.types.numeric.NonNegLong
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+@derive(eqv)
+case class HistoricalSnapshotOrdinalEntries(values: Option[NonEmptyList[HistoricalSnapshotOrdinalEntry]])
+
+case object InvalidRange extends NoStackTrace
+
+@derive(eqv, show)
+case class RangeBoundary(start: NonNegLong, end: NonNegLong)
```

Comment:
1. Is the scope of this class confined to this module? If so shouldn't it be defined inside the companion object?
2. If it is only local to this module, what's the advantage of creating this over using Scala's `NumericRange.Inclusive[Long]`

Reply:
1. Moved `RangeBoundary` to the companion object.
2. `start` and `end` are `NonNegLong`. `NumericRange` does not support `NonNegLong`. Also, I only need the `start` and `end` values, and not the full range.